src/components/PotatoRostisSlides.js

```diff
@@ -0,0 +1,96 @@
+
+import React, { Component } from 'react';
+import items from '../data/potatoRostisSlides.json';
+import {
+  Carousel,
+  CarouselItem,
+  CarouselControl,
+  CarouselIndicators,
+  CarouselCaption
+} from 'reactstrap';
+
+
+
+class PotatoRostisSlides extends Component {
```

Comment:
@Eliasadam You have three slides components for three different recipes. The only difference in these components is the data they're using. So you can just have one component called recipeslides and that just take in different data/recipes 👍

Reply:
yes thats right. ok I will thank you for reviewing my code